code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -18,7 +18,7 @@
 /obj/structure/closet/crate/necropolis/tendril/attackby(obj/item/item, mob/user, params)
 	if(!istype(item, /obj/item/skeleton_key) || spawned_loot)
 		return ..()
-	var/loot = rand(1,20)
+	var/loot = rand(1,21)
```

Comment:
One day I will refactor this because it sucks so bad, no reason for you to do it right now though.